gems/sinatra/CVE-2022-29970.yml

```diff
@@ -0,0 +1,16 @@
+---
+gem: sinatra
+cve: 2022-29970
+ghsa: qp49-3pvw-x4m5
+url: https://github.com/sinatra/sinatra/pull/1683
+title: Sinatra before 2.2.0 does not validate that the expanded path matches public_dir when serving static files.
+date: 2022-05-02
+description: |
+  Sinatra before 2.2.0 does not validate that the expanded path matches
+  public_dir when serving static files.
+cvss_v2: 5.0
+cvss_v3: 7.5
+unaffected_versions:
+- < 2.2.0
```

Comment:
`unaffected_versions` should be removed. [GHSA-qp49-3pvw-x4m5](https://github.com/advisories/GHSA-qp49-3pvw-x4m5) clearly says `< 2.2.0` is affected.

Reply:
that's some bad copy-paste on my side, sorry